### ISSUE TITLE
I wonder if the complete training data is available?

### ISSUE THREAD

**Issue by XYZliang** [gcobs051094/Apex_YOLOv5#4]
I'm an undergraduate student, I've already bailed out of graduate school, and the direction of my advisor is target detection, so I want to get familiar with the content in advance, and I just want to try it with the apex I'm playing with.
The data set is too painful, I wonder if you can provide your training set data, I will get some myself, and then use yolov7 to train to try!

**Comment by gcobs051094** [comment]
About the dataset, you can find it in roboflow. 
Or use the other guys's trained weight to label your own data. 
It will very fast to prepare your training data.
